src/main/scala/com/advancedtelematic/treehub/db/ObjectRepository.scala

```diff
@@ -41,7 +51,7 @@ protected class ObjectRepository()(implicit db: Database, ec: ExecutionContext)
   def update(namespace: Namespace, id: ObjectId, size: Long, status: ObjectStatus): Future[Unit] = {
     val io = findQuery(namespace, id)
       .map { r => r.size -> r.status }
-      .update((size, ObjectStatus.UPLOADED))
+      .update((size, status))
```

Comment:
nice

Reply:
ja. lucky we were only calling this method with that value!

Reply:
;-)